packages/vue/src/stories/getting-started/1. installation.stories.mdx

```diff
@@ -65,6 +68,89 @@ In order to use it in custom projects you need the following webpack loaders: `c
 
 </div>
 
+## Install custom font
+
+StorefrontUI bases on [Raleway](https://fonts.google.com/specimen/Raleway) and [Roboto](https://fonts.google.com/specimen/Roboto) font styles. To include it in your project,
+you need to add the following snippet in the `<head>` of your main `.html` file in a Vue project.
+
+```html
+<link rel="dns-prefetch" href="https://fonts.gstatic.com" />
```

Comment:
I think we should also add snippet for Nuxt/VSF with `vue-meta` as its gonna be majority of our users